### ISSUE TITLE
Adding Code Snippets

### ISSUE THREAD

**Issue by ghost** [ykdojo/csqa#22]
I think you should add an option to share code snippet so that we can share our errors. Thanks!

**Comment by kakasuarez** [comment]
#16 I think you mean something like this? I have already created a pull request for that: #20 .

**Comment by goyalyuval** [comment]
> #16 I think you mean something like this? I have already created a pull request for that: #20 .

Yes I meant this. Sorry I did not see it earlier. You can close the issue.
